packages/@aws-cdk/aws-apprunner-alpha/test/auto-scaling-configuration.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Match, Template } from 'aws-cdk-lib/assertions';
+import * as cdk from 'aws-cdk-lib';
+import { AutoScalingConfiguration } from '../lib';
+
+let stack: cdk.Stack;
+beforeEach(() => {
+  stack = new cdk.Stack();
+});
+
+test.each([
+  ['MyAutoScalingConfiguration'],
+  ['my-autoscaling-configuration_1'],
+])('create an Auto scaling Configuration with all properties (name: %s)', (autoScalingConfigurationName: string) => {
+  // WHEN
+  new AutoScalingConfiguration(stack, 'AutoScalingConfiguration', {
+    autoScalingConfigurationName,
+    maxConcurrency: 150,
+    maxSize: 20,
+    minSize: 5,
+  });
+
+  // THEN
+  Template.fromStack(stack).hasResourceProperties('AWS::AppRunner::AutoScalingConfiguration', {
+    AutoScalingConfigurationName: autoScalingConfigurationName,
+    MaxConcurrency: 150,
+    MaxSize: 20,
+    MinSize: 5,
+  });
+});
+
+test('create an Auto scaling Configuration without all properties', () => {
+  // WHEN
+  new AutoScalingConfiguration(stack, 'AutoScalingConfiguration');
+
+  // THEN
+  Template.fromStack(stack).hasResourceProperties('AWS::AppRunner::AutoScalingConfiguration', {
+    AutoScalingConfigurationName: Match.absent(),
+    MaxConcurrency: Match.absent(),
+    MaxSize: Match.absent(),
+    MinSize: Match.absent(),
+  });
+});
+
+test.each([0, 26])('invalid minSize', (minSize: number) => {
```

Comment:
```suggestion
test.each([-1, 0, 26])('invalid minSize', (minSize: number) => {
```

let's check -1 as well